images/bpfmap-file.proto

```diff
@@ -21,4 +21,5 @@ message bpfmap_file_entry {
 	required string		map_name	= 13;
 	required uint32		ifindex		= 14 [default = 0];
 	optional sint32		mnt_id		= 15 [default = -1];
+	optional uint64		map_extra	= 16;
```

Comment:
Is it zero by default? Should we return a warning if it isn't zero? We have to be sure that we restore mappings properly.

Reply:
Added a warning if it is non-zero.